networkx/algorithms/clique.py

```diff
@@ -902,7 +902,7 @@ def max_weight_clique(G, weight="weight"):
            algoritmo de branch and bound para o problema da clique máxima
            ponderada.  Proceedings of XLVII SBPO 1 (2015).
 
-    .. [2] Warrent, Jeffrey S, Hicks, Illya V.: Combinatorial Branch-and-Bound
+    .. [2] Warren, Jeffrey S, Hicks, Illya V.: Combinatorial Branch-and-Bound
```

Comment:
https://www.semanticscholar.org/paper/Combinatorial-Branch-and-Bound-for-the-Maximum-Set-Warren-Hicks/93bfe2997bbbd34933b3fe44a08f1089bf8b4788